docs/source/directive-resolvers.md

```diff
@@ -0,0 +1,175 @@
+
+## Directive example
```

Comment:
This is the old documentation, just to be clear. The new docs are in `schema-directives.md`.